docs/shrinkwrap.md

```diff
@@ -32,12 +32,13 @@ important that your node_modules is deleted before you do this to be sure to pic
 
 - Install [shonkwrap](https://github.com/skybet/shonkwrap) globally: `npm install -g shonkwrap`. This package attempts
 to remove the from/resolved fields if possible.
+- (Optional) Modify package.json. For example: `npm install --save lodash@4.11.1` or `npm uninstall --save left-pad`
```

Comment:
`npm install --save module@version` will modify the package.json with a npm-shrinkwrap.json present. However, a clean `npm install` will use the exact versions in npm-shrinkwrap.json